Plutarch/Trace.hs

```diff
@@ -34,6 +44,14 @@ ptrace s a = pforce $ ptrace' # s # pdelay a
 ptrace _ a = a
 #endif
 
+-- | Like Haskell's `traceShowId` but for Plutarch
+ptraceShowId :: PShow a => Term s a -> Term s a
+#ifdef Development
+ptraceShowId x = ptrace (pshow x) x
```

Comment:
maybe this function should be called `ptraceShow`

Reply:
Well, it is analogous to Haskell's `traceShowId` - I kept the naming for consistency.